src/pages/signin/ChooseSSOOrMagicCode.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+import ONYXKEYS from '../../ONYXKEYS';
+import Text from '../../components/Text';
+import Button from '../../components/Button';
+import * as Session from '../../libs/actions/Session';
+import ChangeExpensifyLoginLink from './ChangeExpensifyLoginLink';
+import Terms from './Terms';
+import CONST from '../../CONST';
+import ROUTES from '../../ROUTES';
+import Navigation from '../../libs/Navigation/Navigation';
+import useLocalize from '../../hooks/useLocalize';
+import useNetwork from '../../hooks/useNetwork';
+import useWindowDimensions from '../../hooks/useWindowDimensions';
+
+const propTypes = {
+    /* Onyx Props */
+
+    /** The credentials of the logged in person */
+    credentials: PropTypes.shape({
+        /** The email/phone the user logged in with */
+        login: PropTypes.string,
+    }),
+
+    /** The details about the account that the user is signing in with */
+    account: PropTypes.shape({
+        /** Whether or not a sign on form is loading (being submitted) */
+        isLoading: PropTypes.bool,
+
+        /** Form that is being loaded */
+        loadingForm: PropTypes.bool,
```

Comment:
`loadingForm` should be type of `ValueOf<typeof CONST.FORMS>;`?

Reply:
Hmm, that's for typed files I think but maybe `PropTypes.oneOf(CONST.FORMS)`?

Reply:
Actually i think this should be of type string

Reply:
Yes right, I think we can use - PropTypes.oneOf(_.values(CONST.FORMS))